### PR TITLE
sq light intensity + make mutants invis in OpenGL

### PR DESCRIPTION
**Pull request by donnierussellii**
Fix software invisible mutant disappears when near; see commit comment.

Square light intensity to make dark areas dark.

Add shader code to make OpenGL invisible mutants translucent.

### CODE REVIEW

**Comment by donnierussellii** [comment]
Already-pulled gamma factor commit included here for some reason.

**Comment by Interrupt** [comment]
Have a before / after screenshots of how this changes rendering? The OpenGL flow is borked on my dev laptop thanks to Apple, can't link with the 32bit Opengl anymore.

File: src/MacSrc/OpenGL.cc [hunk 1]
```diff
@@ -753,8 +757,12 @@ int opengl_bitmap(grs_bitmap *bm, int n, grs_vertex **vpl, grs_tmap_info *ti) {
         // recalculate it from the offset into the global lighting lookup
         // table.
         light = 1.0 - (ti->clut - grd_screen->ltab) / 4096.0f;
+
+        light *= light;
```
Comment (Interrupt) [inline src/MacSrc/OpenGL.cc]: Instead of squaring the light intensity here, it could be offloaded to the shader instead.

**Comment by donnierussellii** [comment]
I moved the light squaring to the shader.

float light = max(Light * Light, emissive);
vs
float light = max(Light, emissive);

squared_light
unsquared_light

**Comment by donnierussellii** [comment]
Another set of screenshots.

float light = max(Light * Light, emissive);
vs
float light = max(Light, emissive);

squared_light
unsquared_light

**Comment by donnierussellii** [comment]
Screenshot of invisible mutant in OpenGL.

invis_mutant_opengl

**Comment by Interrupt** [comment]
Looks nice! I'm not a fan of how single purpose that `mutant` value in the shader is, usually like shader values to be more general and multi purpose, but until we have a good use case for something else it's probably fine.

**Pull request merged**